validator/sawtooth_validator/journal/consensus/poet/poet_consensus/poet_enclave_factory.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2017 Intel Corporation
```

Comment:
I think you are referring to the old PoetConsensus class, and if that is the case, then no as that code no longer is used.  It is only left in repo until I bring the new 0-8 code into feature parity.